contributing/ci-docs.rst

```diff
@@ -363,38 +363,9 @@ All are currently built from the master branches despite the build names.
 The merge and latest builds for this documentation set are detailed on the
 :doc:`ci-ome-files` page.
 
-Note that because of the bundle nature of these builds, they does not use the
-standard Sphinx configuration described above nor report broken links parsed
-through the warnings plugin as below.
-
-.. _linkcheck_parser:
-
-Linkcheck output parser
-^^^^^^^^^^^^^^^^^^^^^^^
-
-.. _Warnings Plugin: https://wiki.jenkins-ci.org/display/JENKINS/Warnings+Plugin
-
-The :file:`output.txt` file generated by Sphinx ``linkcheck`` builder is
-parsed using the `Warnings Plugin`_. Depending on the nature of the links,
-warnings are generated as described in the following table:
-
-====== ================ ========
-Type   Error code		Priority
-====== ================ ========
-local					High
-broken HTTP Error 404	Normal
-broken Anchor not found Normal
-broken HTTP Error 403	Low
-====== ================ ========
-
-The build is marked as FAILED or UNSTABLE if the number of warnings of a given
-category exceeds a threshold. The table below lists the thresholds used for
-all the documentation builds:
-
-======== ====== ========
-Priority FAILED UNSTABLE
-======== ====== ========
-High	 0
-Normal			0
-Low				10
-======== ====== ========
+Linkchecking
+^^^^^^^^^^^^
+
+Use of the linkcheck output parser was discontinued when the version of Sphinx
```

Comment:
Reading through it for the first time, the immediate question would be "which linkckeck output parser". 
I am unclear whether this section is required at all or if we should drop if and have this policy change recorded via the Git/GitHub history.

Reply:
I initially just dropped it altogether then wondered if I should leave something. I'll delete it again.